Demos/Braintree-Demo/Features/Drop In/BraintreeDemoDropInViewController.m

```diff
@@ -51,6 +64,16 @@ - (void)tappedToShowDropIn {
     dropIn.callToActionText = @"$19 - Subscribe Now";
     dropIn.shouldHideCallToAction = NO;
 
+    if (self.prepopulateDataSwitch.on) {
+        NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+        [dateFormatter setDateFormat:@"MM/yyyy"];
+        [dropIn.cardForm setExpirationDate:[dateFormatter dateFromString: @"12/2018"]];
+        dropIn.cardForm.number = @"4111111111111111";
+        dropIn.cardForm.cvv = @"123";
+        dropIn.cardForm.postalCode = @"12345";
+        dropIn.cardForm.optionalFields = BTUICardFormOptionalFieldsAll;
```

Comment:
Nice! This brings us a big step closer towards easy integration with card.io in Drop in.